yo/app/scripts/directives/transformation.js

```diff
@@ -1132,25 +1132,44 @@ angular.module('dmpApp')
         }
 
         /**
-         * Send a single transformation to the server
+         * This actually sends a Transformation
+         * @param {boolean} persist - Should the data be persisted or just previewed
          */
-        $scope.sendTransformation = function() {
+        function sendTransformationsInternal(persist) {
```

Comment:
sendTransformation was sending a single transformation, not everything. This is no longer the case and DD-953 doesn't mention this.